geometry_msgs/PoseStamped[] grasp_pose_list
---
bool success